src/index.html

```diff
@@ -4,13 +4,60 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <h1 class="heading">Search bar airbnb</h1>
+    <div class="space"></div>
+    <div class="form">
+      <form
+      action="http://www.google.com"
+      method="post"
+      data-qa="big"
+      class="form-search"
+      >
+        <label for="search-bar-one" >
+          <img
+          src="../src/images/Search.svg"
+          alt="magnify glass"
+          class="magnify1">
+          <input
+          type="search"
+          name="search-bar-one"
+          id="searchBarOne"
+          class="search-bar search1"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
```

Comment:
keep your code well-formatted, that's important
make sure to fix such indent issues all over your code